lessons/lesson2.md

```diff
@@ -1,129 +1,102 @@
+- `project.json`: for instance, `projectname-orgId-Production.json` file downloaded from console 
 
-Strong emphasis, aka bold, with **asterisks** or __underscores__.
 
-Combined emphasis with **asterisks and _underscores_**.
+1. Navigate to Adobe I/O console at [https://console.adobe.io](https://console.adobe.io) in your browser and create a project or using your exsiting project 
+2. Select `Add to Project` -> `Add an API` -> `Adobe Services` -> `I/O managemenet API`
+![add-api](assets/add-api.png)
 
-Strikethrough uses two tildes. ~~Scratch this.~~
+3. Follow the steps to configure API, create a new service account (JWT) credential, `config.zip` will be
+downloaded automatically, you will need the private key to generate JWT token.
 
-1. First ordered list item
-2. Another item
-⋅⋅* Unordered sub-list. 
-1. Actual numbers don't matter, just that it's a number
-⋅⋅1. Ordered sub-list
-4. And another item.
+4. Go to `project overview` tab, download project metadata from below `Download` button and get the needed info from this `.json`file, or you can also get these info from `.aio` file in the project folder.
+![download](assets/download.png)
+![projectjson](assets/project-json.png)
 
-⋅⋅⋅You can have properly indented paragraphs within list items. Notice the blank line above, and the leading spaces (at least one, but we'll use three here to also align the raw Markdown).
+5. Get credential details, generate JWT 
+![console](assets/console.png)
 
-⋅⋅⋅To have a line break without a paragraph, you will need to use two trailing spaces.⋅⋅
-⋅⋅⋅Note that this line is separate, but within the same paragraph.⋅⋅
-⋅⋅⋅(This is contrary to the typical GFM line break behaviour, where trailing spaces are not required.)
 
-* Unordered list can use asterisks
-- Or minuses
-+ Or pluses
 
-[I'm an inline-style link](https://www.google.com)
+### Set up project and use CLI to register the app as event provider
+After set up your integration and collect the informations, you can start set up your project 
+* Add `package.json` it contains the list of dependencies, version, reproducible builds, etc.
+* Use CLI to create provider, eventmetadata and webhook registration
 
-[I'm an inline-style link with title](https://www.google.com "Google's Homepage")
-
-[I'm a reference-style link][Arbitrary case-insensitive reference text]
-
-[I'm a relative reference to a repository file](../blob/master/LICENSE)
-
-[You can use numbers for reference-style link definitions][1]
-
-Or leave it empty and use the [link text itself].
-
-URLs and URLs in angle brackets will automatically get turned into links. 
-http://www.example.com or <http://www.example.com> and sometimes 
-example.com (but not on Github, for example).
-
-Some text to show that the reference links can follow later.
-
-[arbitrary case-insensitive reference text]: https://www.mozilla.org
-[1]: http://slashdot.org
-[link text itself]: http://www.reddit.com
-
-![adobe](assets/adobe.png)
+1. `package.json` sample file
```

Comment:
as the events registration, why do we need this package.json?
AFAICS it's only useful if events reg is done in code.

Reply:
done